pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -228,6 +228,9 @@ func manageServiceAccountCABundle(lister corev1listers.ConfigMapLister, client c
 		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.GlobalMachineSpecifiedConfigNamespace, Name: "managed-kube-apiserver-serving-cert-signer"},
 		// for now, include the CA we use to sign CSRs
 		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.OperatorNamespace, Name: "csr-controller-ca"},
+		// include the ca bundle needed to recognize default
+		// certificates generated by cluster-ingress-operator
+		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.GlobalMachineSpecifiedConfigNamespace, Name: "router-ca"},
```

Comment:
@Miciah this looks fine.  I assume the `cluster-ingress-operator` going to create / manage the `router-ca` config map in `openshift-config-managed`?

Reply:
Thanks! It will, yeah, pending https://github.com/openshift/cluster-ingress-operator/pull/110 and https://github.com/openshift/cluster-ingress-operator/pull/111.